internal/commands/secrets/delete_inputs_test.go

```diff
@@ -0,0 +1,98 @@
+package secrets
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/utils/test/assert"
+	"github.com/10gen/realm-cli/internal/utils/test/mock"
+)
+
+func TestResolveDeleteInputs(t *testing.T) {
+	testLen := 7
+	secrets := make([]realm.Secret, testLen)
+	for i := 0; i < testLen; i++ {
+		secrets[i] = realm.Secret{
+			ID:   fmt.Sprintf("secret_id_%d", i),
+			Name: fmt.Sprintf("secret_name_%d", i),
+		}
+	}
+
+	for _, tc := range []struct {
+		description     string
+		selectedSecrets []string
+		expectedOutput  []realm.Secret
+	}{
+		{
+			description:     "select by ID",
+			selectedSecrets: []string{"secret_id_0"},
+			expectedOutput:  []realm.Secret{secrets[0]},
+		},
+		{
+			description:     "select by name",
+			selectedSecrets: []string{"secret_name_2"},
+			expectedOutput:  []realm.Secret{secrets[2]},
+		},
+		{
+			description:     "select by name and ID",
+			selectedSecrets: []string{"secret_name_2", "secret_id_4", "secret_name_1"},
+			expectedOutput:  []realm.Secret{secrets[2], secrets[4], secrets[1]},
+		},
+	} {
+		t.Run("should return the realm.Secret values from deleteInputs: "+tc.description, func(t *testing.T) {
```

Comment:
a note about these `t.Run` strings: try to keep things lowercase and avoid using punctation like `.` or `:` in  (it ends up looking a bit wonky in the resulting `go test` output)

something like `"should return the secrets when specified by"+tc.description` would suffice here